docker/girder.Dockerfile

```diff
@@ -11,6 +12,12 @@ RUN yarn build
 
 FROM girder/girder as runtime
 
+ENV GIRDER_MONGO_URI mongodb://mongo:27017/girder
+ENV GIRDER_ADMIN_USER admin
+ENV GIRDER_ADMIN_PASSWORD viame
+ENV CELERY_BROKER_URL amqp://guest:guest@rabbit/
+ENV BROKER_CONNECTION_TIMEOUT 2
```

Comment:
Why was this added? Do these serve as a default for the variables specified in `docker-compose.yml`? Or if not, how do they interfere with them?

Reply:
Docker images that accept optional environment configuration should typically list them in the dockerfile.  This serves largely as documentation.  

These do serve as defaults if the images are used alone.  They're overridden by the vars supplied in docker-compose.yml